rust/noosphere/src/ffi/noosphere.rs

```diff
@@ -98,26 +91,29 @@ pub fn ns_initialize(
 }
 
 #[ffi_export]
-/// De-allocate a [NoosphereContext]. Note that this will also drop every
-/// [SphereContext] that remains active within the [NoosphereContext].
+/// @memberof ns_noosphere_t
+/// Deallocate a ns_noosphere_t instance.
+///
+/// Note that this will also deallocate every ns_sphere_t that remains
```

Comment:
Notably, this extra new line here creates the fully detailed function on the docs page with extended information. We could annotate other data e.g. @\param, @\return for more clarity as well, which would also create the "full" function def upon generation.